controllers/users.js

```diff
@@ -68,7 +206,7 @@ exports.getUsers = async function (req, res, next) {
       let usersByTags = await models.user.readUsersByTags(tags);
 
       // remap users to a proper format for serializing
-      usersByTags = _.map(usersByTags, (ubt) => {
```

Comment:
I didn't like they were not cosistently used in the project. Probably more functions could be rewritten in this form but I didn't like there were just two of them. If I am am wrong, or generally they can stay.
Let me know @mrkvon.